packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -304,4 +321,12 @@ abstract contract FundingRateApplier is FeePayer {
         // cumulative funding rate multiplier.
         newCumulativeFundingRateMultiplier = currentCumulativeFundingRateMultiplier.mul(unsignedPeriodMultiplier);
     }
+
+    function _getAncillaryData() public view returns (bytes memory) {
```

Comment:
nit: if this is a public method it should probs be with the other funding methods and have natspec/reentrancy guards?

Reply:
it's not. Making it public was accidental!

Reply:
Changing it now :)

Reply:
Done